src/http/cookie.cr

```diff
@@ -104,19 +144,44 @@ module HTTP
       CookieString    = /(?:^|; )#{Regex::CookiePair}/
       SetCookieString = /^#{Regex::CookiePair}(?:;\s*#{Regex::CookieAV})*$/
 
-      def parse_cookies(header)
+      # Yields an `HTTP::Cookie` for each cookie in the header.
+      #
+      # ```crystal
+      # require "http/cookie"
+      #
+      # names = [] of String
+      # HTTP::Cookies::Parser.parse_cookies { |cookie| names << cookie.name }
```

Comment:
Whole module is `:nodoc:`-ed for a reason, so that comment is pointless.
And it's simply wrong anyway.

ditto all of the docs within the `Parser` module.